src/config/ConfigManager.cpp

```diff
@@ -1430,6 +1430,8 @@ void CConfigManager::loadConfigLoadVars() {
     }
 
     Debug::disableStdout = !configValues["debug:enable_stdout_logs"].intValue;
+    if (Debug::disableStdout)
```

Comment:
shouldn't it also be && isFirstLaunch?

Otherwise this will be spammed every time the cfg is reloaded.

Reply:
Oh, I missed that. Should be fixed now.